libcxx/include/__config

```diff
@@ -838,21 +838,30 @@ typedef __char32_t char32_t;
 #  define _LIBCPP_CLANG_DIAGNOSTIC_IGNORED_CXX23_EXTENSION _LIBCPP_CLANG_DIAGNOSTIC_IGNORED("-Wc++2b-extensions")
 #endif
 
+#  if !__has_feature(modules)
```

Comment:
Can you please leave a comment behind explaining why we don't do it for modules?